configure

```diff
@@ -87,28 +88,27 @@ get_version() {
 
 	BITLBEE_VERSION=$REAL_BITLBEE_VERSION
 
-	if [ -d $srcdir/.git ] && type git > /dev/null 2> /dev/null; then
-		timestamp=$(cd $srcdir; git show -s --format=%ci HEAD | sed 's/ .*$//; s/-//g')
-		branch=$(cd $srcdir; git rev-parse --abbrev-ref HEAD)
+	if [ -d "$srcdir/.git" ] && type git > /dev/null 2> /dev/null; then
+		timestamp=$(cd "$srcdir" || exit; git show -s --format=%ci HEAD | sed 's/ .*$//; s/-//g')
```

Comment:
This is [SC2164](https://www.shellcheck.net/wiki/SC2164), but would `git` commands executed in the wrong directory be better? Given the check in line 91, the `exit` in line 92 should only become relevant in race conditions, no?

Reply:
I'm not a fan of adding this sort of thing just to shut up a linter, when there's no functional advantage. In fact, the opposite since this silences the problem rather than giving you a somewhat confusing git error.

Reply:
It shouldn't silence the issue, because `cd` still yells at `STDERR`, but I can revert this indeed. Just to clarify: All occurrences of ` || exit`, that I added?

Reply:
If you let me know, I'm happy to update my pull request.